src/Firehose.Web/Authors/JeffersonBalivo.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Firehose.Web.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+
+namespace Firehose.Web.Authors
+{
+    public class JeffersonBalivo : IAmACommunityMember, IFilterMyBlogPosts
+    {
+        public string FirstName => "Jefferson";
+        public string LastName => "Balivo";
+        public string StateOrRegion => "Jaú, São Paulo, Brazil";
+        public string EmailAddress => "jefferson@balivo.com.br";
+        public string ShortBioOrTagLine => "don't work, \"have fun\" with .NET. I'm \"Line of Business\" software (Desktop, Web and Mobile) specialist. Xamarin enthusiats since was just \"Mono4Android\" and \"MonoTouch\" (~2009).";
```

Comment:
Remember, this short bio will be after your name.

"Jefferson Balivo don't work, have fun with .NET" sounds very strange in English.

Does it really matter to explain what Line of Business means? I would probably just write: "I am a desktop, web and mobile specialist".

"Xamarin enthusiast since it was called Mono for Android and MonoTouch"